config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+const useMocks = process.env.USE_MOCKS === 'true' || !env;
+
+module.exports = {
+  env: env,
+  csp: {
+    imgSrc: ['data:']
+  },
+  useMocks: useMocks,
+  upload: {
+    maxFileSize: '10mb',
+    // if mocks set use file service served up by app otherwise use filevault's port 3000
+    hostname: !useMocks && process.env.FILE_VAULT_URL ?
+      process.env.FILE_VAULT_URL :
+      `http://localhost:${useMocks ? (process.env.PORT || 8080) : 3000}/file`
+  }
+};
```

Comment:
again, I would avoid adding things that are not really related to your PR.

When you actually work on filevault add it then.

You really want a barebones PR

when someone goes back in history it makes it harder to understand otherwise

Reply:
I have removed this for now